typesetting/aegisub.md

```diff
@@ -61,105 +61,115 @@ Here is a list of tools you will want to download:
+*irreversibly ruin the colors of any typesetting,
+dialogue,
+or kfx already in the script*.
+Even worse,
+some video renderers will read this setting from the muxed subtitles
+and render the video to match it.
+
+
+If you are working on a DVD
+or something in Standard Definition,
+you can change this to BT.601 manually in *File \> Script Properties*.
+However, not all Standard Definition video will be BT.601,
+so when in doubt,
+ask the encoder or check the source's
+[MediaInfo][] if they are not available.
 
-[bt601-vs-bt709]: http://blog.maxofs2d.net/post/148346073513/bt601-vs-bt709
 [MediaInfo]: https://mediaarea.net/en/MediaInfo
 
-The "Subtitles Provider".
-
-1.  Just a few years ago, there was a pretty clear consensus on which
-    subtitle renderer to use for anime and softsubs. These days, not so
-    much.
-2.  It used to be that
-    [VSFilter][] was
-    the best renderer around and was the only supported renderer by most
-    fansub groups.
-3.  However, it was eventually replaced
-    with [xy-vsfilter][] and
-    then that was replaced
-    with [xySubFilter][]
-    because VSFilter and xy-vs filter were not keeping up with the
-    demands of subtitles.
-4.  By 2015, however, xySubFilter development had come to a halt and
-    since then, [libass][] has made many
-    improvements both in speed and compatibility with advanced
-    subtitling in part due to contributions from members of the fansub
-    community.
-5.  At the end of the day, which renderer you choose is up to you, but
-    we recommend libass. It is maintained, cross-platform, able to handle
-    most typesetting, and has been integrated into many commercial and
-    open source software products. If you prefer, however, stock Aegisub
-    ships with VSFIlter and line0's builds come with xy-vsfilter.
+
+### The "Subtitles Provider"
+
+The recommended build of Aegisub comes pre-equipped with libass,
+so no manual settings change is needed.
+The following is a brief history of subtitle renderers.
+
+Just a few years ago,
+there was a pretty clear consensus on which
+subtitle renderer to use for anime and softsubs.
+These days, not so much.
+It used to be that [VSFilter][] was the best renderer around
+and was the only supported renderer by most fansub groups.
+However, it was eventually replaced with [xy-vsfilter][]
+and then later replaced with [xySubFilter][]
+because VSFilter and xy-vsfilter were not keeping up
+with the demands of subtitles.
+
+By 2015, however,
+xySubFilter development had come to a halt and since then,
+[libass][] has made many improvements
+both in speed and compatibility with advanced subtitling
+in part due to contributions from members of the fansub community.
+At the end of the day,
+which renderer you choose is up to you,
+but we recommend libass.
+It is maintained,
+cross-platform,
+able to handle most typesetting,
+and has been integrated into many commercial
+and open-source software products.
 
```

Comment:
Maybe add a sentence on which renderer is used where (media player)? VSFilter and forks only work on Windows, for example.